pennylane/wires.py

```diff
@@ -14,48 +14,35 @@
 """
 This module contains the :class:`Wires` class, which takes care of wire bookkeeping.
 """
-from collections.abc import Sequence, Iterable
-import numpy as np
+from collections.abc import Iterable
 from numbers import Number
+import numpy as np
+
+from orderedset import OrderedSet
 
 
 class WireError(Exception):
     """Exception raised by a :class:`~.pennylane.wires.Wire` object when it is unable to process wires."""
 
 
 def _process(wires):
-    """Converts the input to a tuple of numbers or strings."""
+    """ Generates a tuple from whatever nested structure wires might have """
```

Comment:
Might be good to be more specific here (I find this docstring a bit vague). I suppose this function does two things:

1. Validation (ensures wires are unique)
2. Converts any nested structure into a flat tuple